### PR TITLE
init zh-CN translation for `::file-selector-button`

### PR DESCRIPTION
**Pull request by hohoDigidea**

### CODE REVIEW

File: files/zh-cn/web/css/pseudo-elements/index.md [hunk 1]
```diff
@@ -92,6 +124,6 @@ selector::pseudo-element {
   </tbody>
 </table>
 
-## 参见
+## 相关链接
```
Comment (yin1999) [inline files/zh-cn/web/css/pseudo-elements/index.md:127]:
```suggestion
## 参见
```

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+---
+title: '::file-selector-button'
+slug: Web/CSS/::file-selector-button
+page-type: css-pseudo-element
+tags:
+  - CSS
+  - Pseudo-element
+  - Reference
+  - Selector
+browser-compat: css.selectors.file-selector-button
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_file-selector-button/index.md:10]:
```suggestion
```

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+---
+
+{{CSSRef}}
+
+**`::file-selector-button`** [CSS](/zh-CN/docs/Web/CSS) [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements) 代表 [`type="file"`](/zh-CN/docs/Web/HTML/Element/input/file) 的 {{HTMLElement("input") }} 的按钮。
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_file-selector-button/index.md:15]:
```suggestion
**`::file-selector-button`** [CSS](/zh-CN/docs/Web/CSS) [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)代表 [`type="file"`](/zh-CN/docs/Web/HTML/Element/input/file) 的 {{HTMLElement("input") }} 的按钮。
```

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+input[type="file"]::file-selector-button:hover {
+  background-color: #81ecec;
+  border: 2px solid #00cec9;
+}
+```
+
+#### 结果
+
+{{EmbedLiveSample("Basic_example", "100%", 150)}}
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_file-selector-button/index.md:65]:
```suggestion
{{EmbedLiveSample("基础示例", "100%", 150)}}
```

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+请注意 `::file-selector-button` 是一个完整元素，因此与 UA 样式表中的规则相匹配。特别是，字体和颜色不一定会从 `input` 元素继承。
+
+### fallback 示例
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_file-selector-button/index.md:69]:
```suggestion
### 回退示例
```

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+input[type="file"]::file-selector-button:hover {
+  background-color: #81ecec;
+  border: 2px solid #00cec9;
+}
+```
+
+#### 结果
+
+{{EmbedLiveSample("Fallback_example", "100%", 150)}}
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_file-selector-button/index.md:117]:
```suggestion
{{EmbedLiveSample("回退示例", "100%", 150)}}
```

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+## 浏览器兼容性
+
+{{Compat}}
+
+## 相关链接
+
+- [WebKit CSS 扩展](/zh-CN/docs/Web/CSS/WebKit_Extensions)
+- [Microsoft CSS 扩展](/en-US/docs/Web/CSS/Microsoft_Extensions)
+- [File and Directory Entries API](/zh-CN/docs/Web/API/File_and_Directory_Entries_API)
+- [File and Directory Entries API support in Firefox](/zh-CN/docs/Web/API/File_and_Directory_Entries_API/Firefox_support)
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_file-selector-button/index.md:132]:
```suggestion
- [文件与目录条目 API](/zh-CN/docs/Web/API/File_and_Directory_Entries_API)
- [Firefox 对文件与目录条目 API 的支持](/zh-CN/docs/Web/API/File_and_Directory_Entries_API/Firefox_support)
```

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 相关链接
+
+- [WebKit CSS 扩展](/zh-CN/docs/Web/CSS/WebKit_Extensions)
+- [Microsoft CSS 扩展](/en-US/docs/Web/CSS/Microsoft_Extensions)
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_file-selector-button/index.md:130]:
```suggestion
- [Microsoft CSS 扩展](/zh-CN/docs/Web/CSS/Microsoft_Extensions)
```

**Review by yin1999 — COMMENTED** [review]
Thanks for your contribution. For the new comer, it's better to read the [translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md). I think it helps a lot :)

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+## 规范
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 相关链接
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_file-selector-button/index.md:120]:
```suggestion
## 参见
```

**Review by yin1999 — APPROVED** [review]

File: files/zh-cn/web/css/_doublecolon_file-selector-button/index.md (hunk 8 above)
Comment (LIXiangChen) [reply files/zh-cn/web/css/_doublecolon_file-selector-button/index.md]: 这一个链接没有中文版的, 所以我才没改...

Comment (yin1999) [reply files/zh-cn/web/css/_doublecolon_file-selector-button/index.md]: 参考翻译指南，里面有说明

Comment (LIXiangChen) [reply files/zh-cn/web/css/_doublecolon_file-selector-button/index.md]: 哦, 看到了

**Pull request merged**